apps/web/app/(regflow)/sign-up/FlowController.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+import { FormWrapper } from "./SignUpFormStyled";
+
+interface UserInfo {
+  firstName: string;
+  lastName: string;
+  avatar: string;
+}
+
+interface Step {
+  screen: React.ComponentType<any>;
+  props: {
+    userInfo?: string | UserInfo;
+    onNext?: ((value: string) => void) | ((value: UserInfo) => void);
```

Comment:
this could be simplified
```suggestion
    onNext?: (value: string | UserInfo) => void;
```